### PR TITLE
Add wireguard configuration commands

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
This change depends on:
 https://github.com/foundriesio/fioconfig/pull/4/

File: subcommands/config/wireguard.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		if k == "endpoint" {
+			w.Endpoint = v
+		} else if k == "server_address" {
+			w.VpnAddress = v
+		} else if k == "pubkey" {
+			w.PublicKey = v
+		} else if k == "enabled" {
+			w.Enabled = v != "0"
+		} else {
+			fmt.Println("ERROR: Unexcepted client config key: ", k)
```
Comment (ricardosalveti) [inline subcommands/config/wireguard.go:61]: typo: unexpected

File: subcommands/devices/config_wireguard.go [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+		}
+		k := strings.TrimSpace(parts[0])
+		if k == "address" {
+			w.Address = strings.TrimSpace(parts[1])
+		} else if k == "enabled" {
+			w.Enabled = parts[1] != "0"
+		} else if k == "pubkey" {
+			w.PublicKey = strings.TrimSpace(parts[1])
+		} else {
+			fmt.Println("ERROR: Unexcepted client config key: ", k)
```
Comment (ricardosalveti) [inline subcommands/devices/config_wireguard.go:55]: same :-)

**Comment by doanac** [comment]
Here's the server side pieces:

 https://github.com/foundriesio/factory-wireguard-server

File: subcommands/devices/config_wireguard.go (hunk 2 above)
Comment (doanac) [reply subcommands/devices/config_wireguard.go]: force push fix

**Pull request merged**